packages/build/src/steps/get.ts

```diff
@@ -85,7 +87,12 @@ const addCoreSteps = function (steps): CoreStep[] {
 
 // Sort plugin steps by event order.
```

Comment:
Not a blocker and maybe this is a me thing, but improving the comment here with a note on the use of the priority field (or in the code itself) would help make things clearer for the step sorting 👍

Reply:
💯 agreed. I have added a comment in https://github.com/netlify/build/pull/5509/commits/c61913e0299ca82aca92460d79b8eecffa46378e, and improved the types along the way.

Reply:
Actually, I don't think we need a new `priority` field as long as we define the core steps in the right order (i.e. build command needs to happen before applying the deploy configuration API). Adding a new priority field would increase the risk of introducing regressions and would make things harder to understand, because we'd have two implicit sources of priority.

https://github.com/netlify/build/pull/5509/commits/cecad5649aa65ad0fa87fada34d3064baa405230